frontend/src/__tests__/cypress/cypress/e2e/projects/ProjectList.cy.ts

```diff
@@ -44,6 +58,21 @@ describe('Data science projects details', () => {
     cy.url().should('include', '/projects/test-project');
   });
 
+  it('should test url for workbench creation', () => {
+    const mockProject = mockProjectK8sResource({});
+    cy.interceptK8sList(ProjectModel, mockK8sResourceList([mockProject]));
```

Comment:
Can this be used inside a `beforeEach` or `initIntercepts`? I believe this is being executed for every test.